strr-web/composables/useChipFlavour.ts

```diff
@@ -10,11 +10,11 @@ export const useChipFlavour = () => {
     text: tStatuses(translationKey)
   })
 
-  const examinerOrHostStatusMap = (flavour: AlertsFlavourE, key: string) => ({
+  const examinerOrHostStatusMap = (flavour: AlertsFlavourE, commonKey: string, hostSpecificKey?: string) => ({
     alert: flavour,
     text: isExaminer
-      ? tStatuses(`examinerStatuses.${key}`)
-      : tStatuses(`hostStatuses.${key}`)
+      ? tStatuses(`examinerStatuses.${commonKey}`)
+      : tStatuses(`hostStatuses.${commonKey || hostSpecificKey}`)
```

Comment:
`hostSpecificKey` will never be used because `commonKey` is always be there/defined.
Eg. For hostSpecificKey to be used, the line 59 should be: 
`return examinerOrHostStatusMap(AlertsFlavourE.SUCCESS, null, 'provisionalReview')`